felix/docs/config-params.md

```diff
@@ -1366,10 +1366,10 @@ A regular expression that controls which interfaces Felix should attach BPF prog
 | --- | --- |
```

Comment:
Might want to tweak the description in felixconfig.go to make it clear that tunnels should no longer be included :tada: